### ISSUE TITLE
TypeError: process_peering_event() got an unexpected keyword argument 'resource_indexed'

### ISSUE THREAD

**Issue by rtoma** [nolar/kopf#710]
## Long story short

Upgrading kopf to 1.30.0 gives me errors.

## Description

I start my operator locally and it immediately fails. Same failure when running inside k8s.

```
$ kopf run dns_network_policy_operator.py --all-namespaces --peering dns-network-policy-operator --dev
[2021-03-10 20:11:50,537] kopf.reactor.activit [INFO    ] Initial authentication has been initiated.
[2021-03-10 20:11:50,543] kopf.activities.auth [INFO    ] Activity 'login_via_pykube' succeeded.
[2021-03-10 20:11:50,544] kopf.reactor.activit [INFO    ] Initial authentication has finished.
[2021-03-10 20:11:50,840] kopf.reactor.queuein [ERROR   ] Event processing has failed with an unrecoverable error for (clusterkopfpeerings.v1.kopf.dev, '40beea2f-1d36-4ece-8e4d-a24e36dd4cc9').
Traceback (most recent call last):
  File "/Users/rtoma/projects/dns-network-policy-operator/env/lib/python3.9/site-packages/kopf/reactor/queueing.py", line 311, in worker
    await processor(
TypeError: process_peering_event() got an unexpected keyword argument 'resource_indexed'
[2021-03-10 20:11:50,947] kopf.reactor.queuein [ERROR   ] Event processing has failed with an unrecoverable error for (clusterkopfpeerings.v1.kopf.dev, 'd611cf8d-ce78-45e6-b88b-da6ba8263c2c').
Traceback (most recent call last):
  File "/Users/rtoma/projects/dns-network-policy-operator/env/lib/python3.9/site-packages/kopf/reactor/queueing.py", line 311, in worker
    await processor(
TypeError: process_peering_event() got an unexpected keyword argument 'resource_indexed'
[2021-03-10 20:11:50,948] kopf.reactor.orchest [ERROR   ] Peering observer for dns-network-policy-operator@none has failed: Event processing has failed with an unrecoverable error. This seems to be a framework bug. The operator will stop to prevent damage.
Traceback (most recent call last):
  File "/Users/rtoma/projects/dns-network-policy-operator/env/lib/python3.9/site-packages/aiojobs/_scheduler.py", line 138, in _wait_failed
    await task  # should raise exception
  File "/Users/rtoma/projects/dns-network-policy-operator/env/lib/python3.9/site-packages/kopf/reactor/queueing.py", line 311, in worker
    await processor(
TypeError: process_peering_event() got an unexpected keyword argument 'resource_indexed'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/rtoma/projects/dns-network-policy-operator/env/lib/python3.9/site-packages/kopf/utilities/aiotasks.py", line 69, in guard
    await coro
  File "/Users/rtoma/projects/dns-network-policy-operator/env/lib/python3.9/site-packages/kopf/reactor/queueing.py", line 229, in watcher
    raise RuntimeError("Event processing has failed with an unrecoverable error. "
RuntimeError: Event processing has failed with an unrecoverable error. This seems to be a framework bug. The operator will stop to prevent damage.
```

<details><summary>The code snippet to reproduce the issue</summary>

```python
import kopf
```
</details>

<details><summary>The exact command to reproduce the issue</summary>

```bash
kopf run ...
```
</details>

<details><summary>The full output of the command that failed</summary>

```
```
</details>

## Environment

* Kopf version: kopf, version 1.30.0
* Kubernetes version: "v1.19.5+k3s2
* Python version: 3.9.2 
* OS/platform: MacOS and k3s on rpi

<details><summary>Python packages installed</summary>

```
aiohttp==3.7.4.post0
aiojobs==0.3.0
async-timeout==3.0.1
attrs==20.3.0
certifi==2020.12.5
chardet==4.0.0
click==7.1.2
dictdiffer==0.8.1
dnspython==2.1.0
idna==2.10
iso8601==0.1.14
kopf==1.30.0
multidict==5.1.0
pip==21.0.1
pykube-ng==20.10.0
python-json-logger==2.0.1
PyYAML==5.4.1
requests==2.25.1
setuptools==53.0.0
typing-extensions==3.7.4.3
urllib3==1.26.3
wheel==0.36.2
yarl==1.6.3
```
</details>

**Comment by nolar** [comment]
@rtoma I'm terribly sorry about this. Hotfixed in #712, released as [1.30.1](https://github.com/nolar/kopf/releases/tag/1.30.1).

**Comment by rtoma** [comment]
Hi @nolar - thanks for the hotfix. I have deployed it and works fine!
